scripts/pylint_extensions.py

```diff
@@ -795,6 +795,98 @@ def visit_functiondef(self, node):
             self.add_message('function-args-order-cls', node=node)
 
 
+class NewlineCountPrecedingDeclarationChecker(checkers.BaseChecker):
```

Comment:
Please do not create a new checker for this..it's quite similar to the existing single newline checker. Incorporate the logic in the same checker instead of creating a new checker for similar checks.

Reply:
So do you mean that I should combine the logic to `SingleCharAndNewlineAtEOFChecker`? If so then I think the name should change then... To something like `SingleCharFileAndNewlineCountChecker`?